src/components/Messages/MessageButtonGroup.jsx

```diff
@@ -0,0 +1,59 @@
+// React Imports
+import React from 'react';
+// Material UI Imports
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import CreateIcon from '@mui/icons-material/Create';
+import Tab from '@mui/material/Tab';
+import Tabs from '@mui/material/Tabs';
+import useMediaQuery from '@mui/material/useMediaQuery';
+
+const routesArray = [{ label: 'Inbox' }, { label: 'Outbox' }];
+
+/**
+ *  Renders the Message Button Group component for new message, inbox, and outbox
+ *
+ * @param {object} Props - The Props for MessageButtonGroup Component
+ * @param {boolean} Props.showModal - The state to open new message modal
+ * @param {React.Dispatch<React.SetStateAction<boolean>>} Props.setShowModal -
+ * The set function for showModal
+ * @param {string} Props.boxType - The state for inbox or outbox
+ * @param {React.Dispatch<React.SetStateAction<string>>} Props.setBoxType - The
+ * set function for boxType
+ * @returns {React.JSX.Element} - The MessageButtonGroup Component
+ */
+const MessageButtonGroup = ({ showModal, setShowModal, boxType, setBoxType }) => {
+  const isReallySmallScreen = useMediaQuery('(max-width: 480px)');
+
+  return (
+    <Box
+      sx={{
+        display: 'flex',
+        padding: '20px 30px 10px',
+        flexDirection: isReallySmallScreen ? 'column' : 'row',
+        alignItems: isReallySmallScreen ? 'center' : ''
+      }}
+    >
+      <Button
+        variant="contained"
+        onClick={() => setShowModal(!showModal)}
```

Comment:
works this way, but since we only want this button to open the modal - 
`onClick={() => setShowModal(true)}` ?   More predictable/understandable.

Reply:
Think we can do that I'll start a PR on minor revisions and cleanup for v10.